pyfolio/timeseries.py

```diff
@@ -31,6 +28,9 @@
 from .interesting_periods import PERIODS
 
 
+#####################################
```

Comment:
I would either split this or kill the comment. If the distinction is not clear now then I am not sure the comment helps.